projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/vdk_jupyterlab_extension/handlers.py

```diff
@@ -206,9 +216,10 @@ def add_handler(handler, endpoint):
     add_handler(RunJobHandler, "run")
     add_handler(DeleteJobHandler, "delete")
     add_handler(DownloadJobHandler, "download")
-    add_handler(ConvertJobToNotebookHandler, "convertJobToNotebook")
+    add_handler(TransformJobHandler, "convertJobToNotebook")
```

Comment:
Didn't we decide to rename the 'transform job' to 'convert job to notebook'? 
If there was another discussion later, whatever the chosen one is, I'd suggest sticking to it and renaming all the rest occurrences of the other one. As of now, adding this change would mix them and introduce inconsistencies.

Reply:
We discussed and we will be renaming everything to convert